### PR TITLE
Add loop Toggle to Youtube Video Block

### PR DESCRIPTION
**Pull request by jomunker**
This PR adds a loop toggle button to the Youtube Video Block in the Comet Admin. Refer to the Youtube Docs for a list of embed url params: https://developers.google.com/youtube/player_parameters

## Screenshot
image

### CODE REVIEW

**Review by johnnyomair — APPROVED** [review]

**Comment by johnnyomair** [comment]
This PR could go to `main`, as it is a non-breaking change

**Comment by jomunker** [comment]
@johnnyomair I don't think that makes sense since I branched away from next. If `main` is the target branch, all changes from the `next` branch would be merged into `main` with this PR.

**Comment by johnnyomair** [comment]
> @johnnyomair I don't think that makes sense since I branched away from next. If `main` is the target branch, all changes from the `next` branch would be merged into `main` with this PR.

Yes, you should branch away from main for most pull requests. Maybe you can cherry pick your commit.

File: demo/site/src/blocks/YouTubeVideoBlock.tsx [hunk 1]
```diff
@@ -29,10 +29,11 @@ const YouTubeVideoBlock: React.FunctionComponent<PropsWithData<YouTubeVideoBlock
...
     return (
         <sc.VideoContainer heightInPercent={getHeightInPercentForAspectRatio(aspectRatio)}>
+            {/* the playlist parameter is needed so that the video loops. See https://developers.google.com/youtube/player_parameters#loop */}
             <iframe
-                src={`https://www.youtube-nocookie.com/embed/${youtubeIdentifier}?&modestbranding=1&autoplay=${Number(autoplay)}&controls=${Number(
-                    showControls,
-                )}`}
+                src={`https://www.youtube-nocookie.com/embed/${youtubeIdentifier}?&modestbranding=1&autoplay=${Number(autoplay)}${
+                    autoplay ? `&mute=1` : ""
+                }&controls=${Number(showControls)}&loop=${Number(loop)}${loop ? `&playlist=${youtubeIdentifier}` : ""}`}
```
Comment (dkarnutsch) [inline demo/site/src/blocks/YouTubeVideoBlock.tsx:36]: Might be better readable to use URLSearchParams (https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

**Comment by jomunker** [comment]
I created a new branch from main and cherry picked the commit. The new PR is #1206

**Pull request closed without merging**